ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -521,9 +526,18 @@ def _get_columns(
                             data_type_display = column["type"]
                     if parsed_string is None:
                         col_type = ColumnTypeParser.get_column_type(column["type"])
+
                         col_constraint = self._get_column_constraints(
                             column, pk_columns, unique_columns
                         )
+                        if not col_constraint:
+                            if len(pk_columns) > 1:
```

Comment:
@ayush-shah if there is only one pk_columns we should tag that particular key as the primary key